apps/rule-manager/client/src/ts/components/layout/Header.tsx

```diff
@@ -66,17 +66,14 @@ export const Header = () => {
             <Logo />
           </Link>
         </HeaderLogo>
-        {
-        getFeatureSwitchValue('show-tags-page') ?
-          <EuiHeaderLinks>
-            <Link to="/">
-              <EuiHeaderLink isActive={useLocation().pathname === "/"}>Rules</EuiHeaderLink>
-            </Link>
-            <Link to="/tags">
-              <EuiHeaderLink isActive={useLocation().pathname === "/tags"}>Tags</EuiHeaderLink>
-            </Link>
-          </EuiHeaderLinks> : null
-        }
+        <EuiHeaderLinks>
+          <Link to="/">
+            <EuiHeaderLink isActive={useLocation().pathname === "/"}>Rules</EuiHeaderLink>
+          </Link>
+          <Link to="/tags">
+            <EuiHeaderLink isActive={useLocation().pathname === "/tags"}>Tags</EuiHeaderLink>
+          </Link>
+        </EuiHeaderLinks> : null
```

Comment:
😀 

<img width="585" alt="Screenshot 2023-07-17 at 17 11 04" src="https://github.com/guardian/typerighter/assets/7767575/1713c142-845a-4491-9a73-f5adfe675c18">

```suggestion
</EuiHeaderLinks>
```